probe/endpoint/nat.go

```diff
@@ -50,7 +50,10 @@ func toMapping(f Flow) *endpointMapping {
 // ApplyNAT duplicates Nodes in the endpoint topology of a
 // report, based on the NAT table as returns by natTable.
 func (n NATMapper) ApplyNAT(rpt report.Report, scope string) {
-	n.WalkFlows(func(f Flow) {
+	if n.Conntracker == nil {
+		return
+	}
```

Comment:
I'd put this test in reporter.go; it creating the conntracker fails, don't both to create the nat mapper.

There is already a test for the natmapper being nil: https://github.com/weaveworks/scope/blob/master/probe/endpoint/reporter.go#L142